script/convert_mesh.py

```diff
@@ -113,6 +117,26 @@ def main():
 
         mesh = domain.mesh
 
+    if options.merge:
+        coors = mesh.coors
+        kdtree = cKDTree(coors)
+        vmap = nm.asarray(list(kdtree.query_pairs(1e-9)))
+
+        if len(vmap) > 0:
+            remap = nm.ones((coors.shape[0],), dtype=nm.int32)
+            remap[vmap[:, 1]] = -1
+            vidxs = nm.where(remap > 0)[0]
+            remap[vidxs] = nm.arange(len(vidxs))
+            new_coors = coors[vidxs, :]
+            remap[vmap[:, 1]] = remap[vmap[:, 0]]
+            desc = mesh.descs[0]
+            new_conn = remap[mesh.get_conn(desc)]
+            new_ngroups = mesh.cmesh.vertex_groups[vidxs]
+
+            mesh = Mesh.from_data(mesh.name + '_merged',
+                                  new_coors, new_ngroups,
+                                  [new_conn], [mesh.cmesh.cell_groups], [desc])
+
```

Comment:
Yes it works. I use it now, but I would like to reimplement fix_double_nodes() using cKDTree.